crates/swc_common/src/source_map.rs

```diff
@@ -1281,12 +1281,17 @@ impl SourceMap {
             let mut col = max(chpos, linechpos) - min(chpos, linechpos);
 
             if let Some(orig) = &orig {
-                if let Some(token) = orig.lookup_token(line, col) {
+                if let Some(token) = orig
+                    .tokens()
+                    .find(|token| token.get_src_line() == line - 1 && token.get_src_col() == col)
```

Comment:
Isn't this going to be prohibitively expensive?

Can you leave a comment on why this is being done?